test/Sentry.Tests/HubTests.verify.cs

```diff
@@ -53,7 +53,8 @@ await Verify(worker.Envelopes)
                     _.DebugFile.Contains("Xunit.SkippableFact") ||
                     _.DebugFile.Contains("xunit.runner") ||
                     _.DebugFile.Contains("JetBrains.ReSharper.TestRunner") ||
-                    _.DebugFile.Contains("Microsoft.TestPlatform")
+                    _.DebugFile.Contains("Microsoft.TestPlatform") ||
+                    _.DebugFile.Contains("Microsoft.VisualStudio.TestPlatform.Common.pdb")
```

Comment:
🔥

Reply:
hmm, I haven't noticed that. let's see if it passes now